src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsInfo.cs

```diff
@@ -138,19 +137,22 @@ public void SetTensorName(int idx, string name)
         }
 
         /// <summary>
-        /// Gets the tensor name with given index.
+        /// Calculates the byte size of tensor data.
         /// </summary>
-        /// <param name="idx">The index of the tensor.</param>
-        /// <returns>The tensor name.</returns>
-        /// <exception cref="IndexOutOfRangeException">Thrown when the index is greater than the number of Tensor.</exception>
+        /// <param name="idx">The index of the tensor information in the list</param>
+        /// <returns>The byte size of tensor</returns>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
         /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
-        /// <since_tizen> 6 </since_tizen>
-        public string GetTensorName(int idx)
+        /// <since_tizen> 7 </since_tizen>
+        public int GetTensorSize(int idx)
```

Comment:
How about provide a method to get TensorInfo object instead of this method?

``` c#
 public TensorInfo this[int idx];


 for (int = 0; i < tensorsInfo.Count; i++)
      tensorsInfo.GetTensorSize(i);

vs

 for (int = 0; i < tensorsInfo.Count; i++)
      tensorsInfo[i].GetSize();



```

Reply:
Refer [this link](https://github.com/Samsung/TizenFX/pull/1154/files/cd49132137f5c954be0e53f4e731f12b8b49a592#r358032811).